src/main/java/bookstore/controller/BookController.java

```diff
@@ -33,37 +34,42 @@ public class BookController {
 
     @GetMapping
     @ResponseStatus(HttpStatus.OK)
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
```suggestion
    @PreAuthorize("hasRole('USER')")
```
you can avoid ROLE_ prefix (can be applied to other endpoints)